components/notification/typings.ts

```diff
@@ -27,7 +27,7 @@ export interface NzNotificationDataOptions<T = {}> {
 
 export interface NzNotificationData {
   title?: string | TemplateRef<void>;
-  content?: string | TemplateRef<void>;
+  content?: string | TemplateRef<void | { $implicit: NzNotificationComponent; data: NzSafeAny }>;
```

Comment:
I think void is unnecessary here:
```suggestion
  content?: string | TemplateRef<{ $implicit: NzNotificationComponent; data: NzSafeAny }>;
```

Reply:
Yes, understanding your point :), i did that to ensure a compatibility with the previous version of Ng Zorro. If we remove this vois type we will have a breaking change.

Reply:
I got it :)

Reply:
So Will I remove the void type and say this PR have a breaking change ?